corona-warn-app/cwa-app-android
# SRS Server Api (EXPOSUREAPP-14188, 14189)

## Test 
- Switch to `Mock-Cloud` env
- `Test Menu -> Submission Test Options -> SRS Submission`
- Try submit 
- To disable device time check -> use Switch

## TODO 

## Tickets
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14188
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14189
